internal/model/model.go

```diff
@@ -40,11 +40,32 @@ const (
 	helpStateFull
 )
 
+type sortType int
+
+const (
+    sortStateByName       sortType = iota 
+    sortStateByPercentage 
+    //sortStateByStatements 
+)
+
+type sortOrder bool
+
+const (
+    ASC      sortOrder = true
+    DSC      sortOrder = false
```

Comment:
No need to make these exported if everything else is not exported (use lowercase letters)